clients/admin-ui/src/features/common/Layout.tsx

```diff
@@ -24,14 +24,25 @@ const Layout = ({
 }) => {
   const features = useFeatures();
   const router = useRouter();
-  const { data: activeMessagingProvider } =
-    useGetActiveMessagingProviderQuery();
-  const { data: activeStorage } = useGetActiveStorageQuery();
-  const showConfigurationNotificationBanner =
-    (!activeStorage || !activeMessagingProvider) &&
-    features.flags.privacyRequestsConfiguration &&
-    (router.pathname === "/privacy-requests" ||
-      router.pathname === "/datastore-connection");
+  const showConfigurationNotificationBanner = () => {
+    if (
+      router.pathname === "/privacy-requests" ||
+      router.pathname === "/datastore-connection"
+    ) {
+      const { data: activeMessagingProvider } =
```

Comment:
This and the call below were being called on every page the `Layout` was being used. This allows it to be called only on the pages that need to be aware of any existing data.

Reply:
That being said, a little lost on how to resolve the linting errors below 🤷‍♀️

Reply:
I would suggest using a useEffect hook for showing the configuration notification banner along with the privacyRequestConfiguration feature flag for checking whether or not to invoke your hooks. Unfortunately, you cannot use the `useGetActiveMessagingProviderQuery` and `useGetActiveStorageQuery` hooks directly in a given function.